content/index.md

```diff
@@ -23,3 +23,32 @@ Ionide is an organization which builds **high quality, cross platform developer
 * Focus on UX, everything should Just Work™
 * Contribute to the general F# OSS ecosystem
 
+
+### Toolchain
+
+
+```mermaid
+---
+title: Ionide/FsAutoComplete toolchain
+---
+
+erDiagram
+    Ionide-ProjInfo ||--|| MsBuild : "To get information out of msbuild"
+
+    FsAutocomplete ||--|| Ionide-ProjInfo : "To get information for the F# Compiler"
+    FsAutocomplete ||--|| Ionide-LanguageServerProtocol  : "To communicate with an LSPClient"
+    FsAutocomplete ||--|| Ionide-FSharpSDKAnalyzers : "To run analyzers"
+    FsAutocomplete ||--|| "FSharp.Compiler.Service" : "To get type information, errors, warnings, etc."
+    FsAutocomplete ||--|| Fantomas : "To format code."
+    FsAutocomplete ||--|| FsharpLint : "To lint code."
+    
+    "ionide-vscode-fsharp" ||--||  FsAutocomplete : "Uses LSP to communicate to"
+    "ionide-vscode-fsharp" ||--||  Ionide-FsGrammer : "To colorize"
```

Comment:
```suggestion
    "ionide-vscode-fsharp" ||--||  Ionide-FsGrammar : "To colorize"
```

Reply:
I also wonder if it's interesting to note that tools like highlight.js use Ionide-FsGrammar's textmate grammar to work.

Reply:
I don't know much about it so any more docs on that repo would be great!